docs/middleware/android-sdk.md

```diff
@@ -1,190 +1,126 @@
-# Sourcing Entity - Android SDK
-FinBox Lending SDK is a one stop solution to seamlessly onboard users to a loan application journey.
+# FinBox Lending: Android
 
-## Modules
-The SDK is broken down into multiple modules which can be customizable as per requirements. Complete SDK has following modules:
-1. Onboarding
-2. Loan dashboard
+FinBox Lending SDK is a drop-in module which can add a digital lending journey to any mobile application
+The SDK has following modules:
```

Comment:
Add a full stop before this, and add another space, so that it becomes a different paragraph. You can check the deploy preview.